### PR TITLE
endoc-425 deadlinkchecker cleanup

### PR DESCRIPTION
**Pull request by jyunmitch**

### CODE REVIEW

File: vuepress/docs/v6.3.2/tutorials/devops/change-default-datasource.md [hunk 1]
```diff
@@ -22,11 +22,11 @@ Create a project, ideally in Git repository, and put the following files in the
 It is essential for your custom Docker image to use a file from one of the official Entando Docker base images.
 
 If you intend to use provide your own EAP based image, please use this 
-[standalone-openshift.xml](https://github.com/entando/entando-docker-base-images/blob/master/entando-eap71-clustered-base/standalone-openshift.xml)
+[standalone-openshift.xml](https://github.com/entando/entando-app-engine-base/blob/develop/eap/standalone-openshift.xml)
```
Comment (jyunmitch) [inline vuepress/docs/v6.3.2/tutorials/devops/change-default-datasource.md:25]: @nshaw, please check that this and the following two links to repos are the correct ones.

Comment (nshaw) [reply vuepress/docs/v6.3.2/tutorials/devops/change-default-datasource.md:25]: The first 2 look fine.

File: vuepress/docs/v6.3.2/tutorials/devops/change-default-datasource.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ may have unintended consequences.
 #### 1.2. Any modules that may be required for your datasource or other connection resource
 If the resource you need to connect to requires some custom classes, such as JDBC drivers, please add these as  
 a module to Wildfly/EAP. As an example, you can look at our
```
Comment (nshaw) [inline vuepress/docs/v6.3.2/tutorials/devops/change-default-datasource.md:37]: Let's remove the "As an example" sentence plus the following sentence. It's not an equivalent resource for the example here and I don't see an obvious one off hand.

**Review by nshaw — CHANGES_REQUESTED** [review]
A couple of comments. Plus I think the data source file may also need to be changed in next?

**Review by nshaw — APPROVED** [review]

**Pull request merged**